cardano-api/src/Cardano/Api/Governance/Poll.hs

```diff
@@ -121,14 +169,48 @@ instance HasTypeProxy GovernancePollAnswer where
 
 instance SignableRepresentation GovernancePollAnswer where
     getSignableRepresentation =
-      error "not implemented"
+      hashToBytes . hashWith @(HASH StandardCrypto) (serialiseToCBOR . asTxMetadata)
+
+instance AsTxMetadata GovernancePollAnswer where
+    asTxMetadata GovernancePollAnswer{govAnsPoll, govAnsChoice} =
+      makeTransactionMetadata $ Map.fromList
+        [ ( metadataLabel
+          , TxMetaMap $
+           [ ( TxMetaNumber 2, TxMetaBytes (serialiseToRawBytes govAnsPoll) )
+           , ( TxMetaNumber 3, TxMetaNumber (toInteger govAnsChoice) )
+           ]
+          )
+        ]
 
 instance SerialiseAsCBOR GovernancePollAnswer where
```

Comment:
It would be good to have some round tripping property tests.

There are some in `Test.Cardano.Api.Typed.CBOR`.

Will need new generators for this data type.

Reply:
Done. See:

```
  roundtrip GovernancePoll CBOR:                        OK (0.09s)
      ✓ roundtrip GovernancePoll CBOR passed 100 tests.
  roundtrip GovernancePollAnswer CBOR:                  OK
      ✓ roundtrip GovernancePollAnswer CBOR passed 100 tests.
  roundtrip GovernancePollWitness CBOR:                 OK (0.01s)
      ✓ roundtrip GovernancePollWitness CBOR passed 100 tests.
```